pkg/operator/sync_test.go

```diff
@@ -7,6 +7,7 @@ import (
 	"time"
 
 	. "github.com/onsi/gomega"
+	v1 "github.com/openshift/api/config/v1"
```

Comment:
nit: could you please split these import blocks? :)

Reply:
sure, done